### DISCUSSION TITLE
terminal input lxqt-session and then have some error

### DISCUSSION THREAD

**Discussion by digger-yu** [lxqt/lxqt#2470] [General]
##### Current Behavior
in terminal input lxqt-session
and then have some error 
![f975d3e2c7a90db322945c019db6f57](https://github.com/lxqt/lxqt/assets/55081697/f17d3e2b-d96b-448f-8137-d63b59194d4b)

##### Steps to Reproduce (for bugs)

1. open terminal , input lxqt-session and enter
2. It happens every time. Need any logs?
3. I'm not quite sure if this is normal or a problem, but it's never good to have a lot of buggy popups!
4. 

##### System Information
```                                                        --->
root@test:~# cat /etc/os-release
PRETTY_NAME="Ubuntu 22.04.3 LTS"
NAME="Ubuntu"
VERSION_ID="22.04"
VERSION="22.04.3 LTS (Jammy Jellyfish)"
VERSION_CODENAME=jammy
ID=ubuntu
ID_LIKE=debian
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
UBUNTU_CODENAME=jammy
root@test:~# cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=22.04
DISTRIB_CODENAME=jammy
DISTRIB_DESCRIPTION="Ubuntu 22.04.3 LTS"
root@test:~# uname -a
Linux test 5.15.0-83-generic #92-Ubuntu SMP Mon Aug 14 09:30:42 UTC 2023 x86_64 x86_64 x86_64 GNU/Linux
root@test:~#

i use lxdm and xfwm4 and lxqt 1.3.0
```
![image](https://github.com/lxqt/lxqt/assets/55081697/5d8c65a5-30e6-4134-a4fc-e25345bf13dc)

**Comment by stefonarch** [comment]
`lxqt-session` is already running, of course you get errors with duplicate modules, as the terminal prints in your screen. It is not meant to be executed directly. What do you want to achieve? Maybe you mean: `lxqt-config-session`, but just open "Session settings" from the menu or runner.

Transferring to "Discussions".

**Reply by digger-yu** [reply]
I see, of course it's fine to just report the error, but wouldn't it be better to say it's already running? For the average user
